src/runners/runner.js

```diff
@@ -0,0 +1,60 @@
+
+const uuid4 = require('uuid4');
+const {
+    utils,
+    eventInterface,
+    event: serverlessEvent,
+    errorCode,
+    tracer,
+} = require('epsagon');
+
+/**
+ * Create and initialize a new serverless event in the epsagon format.
+ * @param {string} resourceType resourceType name
+ * @param {string} name Event name
+ * @param {string} operation Operation name
+ * @param {string} origin Origin name (optional)
+ * @returns {Object} Object with dnsEvent and event start time.
+ */
+const initializeEvent = (resourceType, name, operation, origin) => {
```

Comment:
this looks duplicated with `epsgon-node`

Reply:
you can require specific files from the package for now

Reply:
I know it, It's duplicate because I don't want someone will change the functions sometime in Epsagon-node and it will directly affect here.
If we had a common library, I would agree.